app/src/main/java/eu/laramartin/medsreminder/common/DialogsUtility.java

```diff
@@ -28,4 +32,52 @@ public void onClick(DialogInterface dialog, int id) {
         AlertDialog alertDialog = builder.create();
         alertDialog.show();
     }
+
+    public static void showInviteFriendDialog(Context context) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(context);
+        builder.setTitle(R.string.dialog_permissions_title);
+        final EditText input = new EditText(context);
+        input.setInputType(InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS);
+        builder.setView(input);
+
+        builder.setMessage(R.string.dialog_give_permissions_to_friend);
+        builder.setPositiveButton(R.string.dialog_accept, new DialogInterface.OnClickListener() {
```

Comment:
Check if calls can be chained rather than on the builder object. Example:
```
new Builder(..)
    .setTitle(...)
    .setInputType(...)
    // etc
    .create()
    .show()
```